cpg_workflows/stages/large_cohort.py

```diff
@@ -20,8 +20,8 @@ class Combiner(CohortStage):
     def expected_outputs(self, cohort: Cohort) -> Path:
         if vds_version := get_config()['workflow'].get('vds_version'):
             vds_version = slugify(vds_version)
-            if not vds_version.startswith('v'):
-                vds_version = f'v{vds_version}'
+            # if not vds_version.startswith('v'):
```

Comment:
Could we please revert this in this PR? All good if we end up going for this change, but it's outside the scope and should be introduced in a separate PR :)